packages/generate-EN10168-pdf-template/utils/translate.ts

```diff
@@ -0,0 +1,22 @@
+import * as FR from './FR.json';
+import * as EN from './EN.json';
+import * as DE from './DE.json';
+import * as PL from './PL.json';
+
+import {CertificateLanguages} from "../src/types/schemaTypes"
+
+export class Translate {
+  private languages: CertificateLanguages;
+  private translation;
+
+  constructor (_languages: CertificateLanguages){
+  this.languages = _languages;
+  this.translation = { FR, EN, DE, PL };
+  }
+
+  translate(phrase: string, group: 'certificateFields' | 'certificateGroups' | 'otherFields'): string{
+    if (group === 'certificateFields') return `${phrase} ${this.languages.map(language =>
+      this.translation[language][group][phrase]).join(' / ')}`;
+    return this.languages.map(language => this.translation[language][group][phrase]).join(' / ')
+  }
+}
```

Comment:
Here the idea is load the translations from the schema repository.

Please have a look at this function in generate-html module
```ts
const getTranslations = async (certificateLanguages: string[], schemaConfig: SchemaConfig): Promise<Translations> => {
  const translationsArray = await Promise.all(
    certificateLanguages.map(async (lang) => {
      const filePath = getRefSchemaUrl(schemaConfig, `${lang}.json`).href;
      return { [lang]: (await loadExternalFile(filePath, 'json')) as any };
    }),
  );

  return translationsArray.reduce((acc, translation) => {
    const [key] = Object.keys(translation);
    acc[key] = translation[key];
    return acc;
  }, {});
};
```